Content.Shared/Radiation/SharedRadiationStorm.cs

```diff
@@ -10,6 +10,7 @@ public abstract class SharedRadiationPulseComponent : Component
     {
         public override string Name => "RadiationPulse";
 
+        public virtual float RadsPerSecondBase { get; set; }
```

Comment:
I don't think the client makes use of `RadsPerSecondBase`, so you can just leave that server-side?